docs/maps/reverse-geocoding-tutorial.asciidoc

```diff
@@ -0,0 +1,180 @@
+[role="xpack"]
+[[reverse-geocoding-tutorial]]
+== Map custom regions with reverse geocoding
+
+*Maps* comes with https://maps.elastic.co/#file[predefined regions] that allow you to quickly visualize regions by metrics. *Maps* also offers the ability to map your own regions. You can use any region data you'd like, as long as your source data contains an identifier for the corresponding region. 
+
+But how can you map regions when your source data does not contain a region identifier? This is where reverse geocoding comes in. Reverse geocoding is the process of assigning a region identifer to a feature based on its location.
+
+In this tutorial, you’ll use reverse geocoding to visualize United States Census Bureau Combined Statistical Area (CSA) regions by web traffic.
+
+You’ll learn to:
+
+- Upload custom regions.
+- Reverse geocode with the {es} {ref}/enrich-processor.html[enrich processor].
+- Create a map and visualize CSA regions by web traffic.
+
+When you complete this tutorial, you’ll have a map that looks like this:
+
+[role="screenshot"]
+image::maps/images/reverse-geocoding-tutorial/csa_regions_by_web_traffic.png[]
+
+
+[float]
+=== Step 1: Index web traffic data
+GeoIP is a common way of transforming an IP address to a longitude and latitude. GeoIP is roughly accurate on the city level globally and neighborhood level in selected countries. It’s not as good as an actual GPS location from your phone, but it’s much more precise than just a country, state, or province. Use {ref}/geoip-processor.html[GeoIP processor] to transform an IP adderess into a {ref}/geo-point.html[geo_point] field.
```

Comment:
```suggestion
GeoIP is a common way of transforming an IP address to a longitude and latitude. GeoIP is roughly accurate on the city level globally and neighborhood level in selected countries. It’s not as good as an actual GPS location from your phone, but it’s much more precise than just a country, state, or province. Use {ref}/geoip-processor.html[GeoIP processor] to transform an IP address into a {ref}/geo-point.html[geo_point] field.
```

Reply:
Use GeoIP processor to transform an IP address into a {ref}/geo-point.html[geo_point] field.

This sentence confuses me.  It's telling me to do something, but the follow up steps are to download sample data. When the does the transformation of the IP address actually happen?

Reply:
Great feedback. I have restructured this part to indicate use of geoIp is optional and no needed in the tutorial.